extensions/big-data-cluster/src/bigDataCluster/dialog/bdcDashboardModel.ts

```diff
@@ -94,6 +94,28 @@ export class BdcDashboardModel {
 	}
 }
 
+/**
+ * Retrieves the troubleshoot book URL for the specified service, defaulting to the BDC
+ * troubleshoot notebook if the service name is unknown.
+ * @param service The service name to get the troubleshoot notebook URL for
+ */
+export function getTroubleshootNotebookUrl(service: string): string {
```

Comment:
This would be a little cleaner and a little more extendable with a service -> string map maybe (?)

Reply:
Any specific benefits you had in mind? I'm all for that generally but given that this is only used here I'm not really sure creating an extra object is going to help anything.

Reply:
Nothing really, I guess just a style :P